R/Tendril.R

```diff
@@ -207,24 +251,24 @@ Tendril <- function(mydata,
     tendril.retval <- Tendril.stat(tendril.retval, suppress_warnings)
 
     #format data: assign column names and remove unwanted columns
-    data <- data.frame(Unique.Subject.Identifier = as.character(tendril.retval$data$Unique.Subject.Identifier),
-                       Terms = tendril.retval$data$Terms,
-                       Treat = tendril.retval$data$Treat,
-                       StartDay = tendril.retval$data$StartDay,
-                       rot.factor = tendril.retval$data$rot.factor,
-                       x = tendril.retval$data$x,
-                       y = tendril.retval$data$y,
-                       Tag = tendril.retval$data$Tag,
-                       p = tendril.retval$data$p,
-                       p.adj = tendril.retval$data$p.adj,
-                       fish = tendril.retval$data$fish,
-                       rdiff = tendril.retval$data$rdiff,
-                       RR = tendril.retval$data$RR,
-                       OR = tendril.retval$data$OR,
-                       FDR.tot = tendril.retval$data$FDR.tot,
-                       TermsCount = tendril.retval$data$TermsCount
-    )
-    tendril.retval$data <- data
+#    data <- data.frame(Unique.Subject.Identifier = as.character(tendril.retval$data$Unique.Subject.Identifier),
```

Comment:
Don't comment out code. We can always recover code by going back. If it's no longer relevant, remove it.